README.md

```diff
@@ -5,11 +5,10 @@
 # 📗 Table of Contents
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Hello @HAMMAS-SALEEM👋!
Thank you for your great review I will consider these points.

Thank you 😊